python/tvm/topi/arm_cpu/pooling.py

```diff
@@ -17,9 +17,94 @@
 # pylint: disable=invalid-name, unused-variable
 """Schedule for pooling operators"""
 
-from .mprofile.dsp.pool import pool_dsp_schedule
+import logging
+from tvm import topi, te
```

Comment:
is it possible not to self-import `tvm.topi` inside `tvm.topi`? there might be several existing cases, but it might lead to cyclic import at times

Reply:
Thanks @junrushao, this should now be fixed